examples/petset/README.md

```diff
@@ -0,0 +1,183 @@
+    will step down from the PRIMARY role if it had it. In this case, reads might
+    be served by a SECONDARY, and writes will fail. As soon as one more member
+    is back up, an election will pick a new PRIMARY and reads and writes will
+    succeed normally.
+
+4. All members are down
+
+    In this extreme case, obviously reads and writes will fail. Once two or more
+    members are back up, an election will reestablish the replica set to have a
+    PRIMARY and a SECONDARY, such that reads and writes will succeed normally.
+
+**Note**: for production usage, you should maintain as much separation between
+members as possible. It is recommended to use one or more of the
+[node selection features](http://kubernetes.io/docs/user-guide/node-selection/)
+to schedule PetSet pods into different nodes, and to provide them storage backed
+by independent volumes.
+
+### Scaling Up
+
+MongoDB recommends an odd number of members in a replica set. An admin may
+decide to have, for instance, 5 members in the replica set. Given that there are
+sufficient available persistent volumes, or a dynamic storage provisioner is
+present, scaling up is done with the `oc scale` command:
+
+```bash
+oc scale --replicas=5 petset/mongodb
+```
+
+New pods (containers) are created and they connect to the replica set, updating
+its configuration.
+
+With five members, the scenarios described in the previous section should work
+similarly, though now there is an added resilience to tolerate up to 2 members
+being simultaneously unavailable.
+
+**Note**: scaling up an existing database might require manual intervention. If
+the database size is greater than the `oplogSize` configuration, a manual
+initial sync of the new members will be required. Please consult the MongoDB
+replication manual for more information.
+
+### Scaling Down
+
+An admin may decide to scale down a replica set to save resources or for any
+other reason. For instance, it is possible to go from 5 to 3 members, or from 3
+to 1 member.
+
+While scaling up might be done without manual intervention when the
+preconditions are met (storage availability, size of existing database and
+`oplogSize`), scaling down always require manual intervention.
+
+To scaling down, start with setting the new number of replicas, e.g.:
+
+```bash
+oc scale --replicas=3 petset/mongodb
+```
+
+Note that if the new number of replicas still constitutes a majority of the
+previous number, it is guaranteed that the replica set may elect a new PRIMARY
+in case one of the pods that was deleted had that role. For example, that is the
+case when going from 5 to 3 members.
+
+On the other hand, scaling down to a lower number will temporarily render the
+replica set to have only SECONDARY members and be in read-only mode. That would
+be the case when scaling from 5 down to 1 member.
+
+The next step is to update the replica set configuration to
+[remove members](https://docs.mongodb.com/manual/tutorial/remove-replica-set-member/)
```

Comment:
> It's already there: "On the other hand, scaling down to a lower number will temporarily render the replica set to have only SECONDARY members and be in read-only mode. That would be the case when scaling from 5 down to 1 member."

Yes. And in next paragraph there is that user have to update config manually. But it is not possible in this case!
Also I think that "replica set to have only SECONDARY members and be in read-only mode" might not be true too. It depends on what is a official OpenShift way how user applications should address replicaset?